### PR TITLE
Add remaining MBF21 action pointers

### PR DESCRIPTION
**Pull request by XaserAcheron**
Just a handful of new MBF21 action pointers. This rounds out the full planned set. :D

### CODE REVIEW

**Comment by XaserAcheron** [comment]
A couple of functions got a new arg, so I pushed a quick update commit that adds 'em -- nothin' fancy.

File: src/main/java/net/mtrop/doom/tools/decohack/data/DEHActionPointer.java [hunk 1]
```diff
@@ -114,6 +114,18 @@
...
+	CLEARTRACER         (-1, "ClearTracer", DEHActionPointerType.MBF21 ),
+	JUMPIFHEALTHBELOW   (-1, "JumpIfHealthBelow", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFTARGETINSIGHT (-1, "JumpIfTargetInSight", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.ANGLE_FIXED ),
+	JUMPIFTARGETCLOSER  (-1, "JumpIfTargetCloser", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFTRACERINSIGHT (-1, "JumpIfTracerInSight", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.ANGLE_FIXED ),
+	JUMPIFTRACERCLOSER  (-1, "JumpIfTracerCloser", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	JUMPIFFLAGSSET      (-1, "JumpIfFlagsSet", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	ADDFLAGS            (-1, "AddFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
+	REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
```
Comment (MTrop) [inline src/main/java/net/mtrop/doom/tools/decohack/data/DEHActionPointer.java:128]: Probably should have caught this in earlier PRs, But could you just remove the spaces in between the last parameter and the closing parenthesis on lines 95-138? A la:
```
REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT ),
```
to:
```
REMOVEFLAGS         (-1, "RemoveFlags", DEHActionPointerType.MBF21, DEHActionPointerParam.UINT, DEHActionPointerParam.UINT),
```

**Review by MTrop — CHANGES_REQUESTED** [review]
Looks good, just one tiny style change to keep things in line with the rest of the codebase.

File: src/main/java/net/mtrop/doom/tools/decohack/data/DEHActionPointer.java (hunk 1 above)
Comment (XaserAcheron) [reply src/main/java/net/mtrop/doom/tools/decohack/data/DEHActionPointer.java]: Will do! :)

**Review by MTrop — APPROVED** [review]
Approved.

**Pull request merged**